app/main/views/marketplace.py

```diff
@@ -93,6 +110,53 @@ def format_metrics(columns, column_name):
                     values[k].append(column[k])
                 else:
                     values[k].append(None)
+
+        # order seller type values from largest to smallest
+        if column_name == 'seller_type':
+            type_data = {x['seller_type']: x['count'] for x in columns}
+            ordered_types = reversed(sorted(type_data, key=type_data.get))
+
+            new_values = {'count': []}
+            for seller_type in ordered_types:
+                new_values['count'].append(type_data[seller_type])
+            column_names = type_data.keys()
+            values = new_values
+
+        if column_name == 'domain':
+            metrics = values.keys()
+            new_values = {}
+            domain_order = ['Strategy and Policy',
```

Comment:
is it possible to use an ordereddict or similar here instead of hardcoding the names 3 times?